tab_sponsorship.md

```diff
@@ -41,6 +41,8 @@ As it turns out, writing a book to a professional standard is a challenging task
 
 100% of the funds raised go directly into the project budget and will be used to fund production of the final release, including:
 
-Editing and proofreading by professional editors
-Graphic design and layout
-Any leftover funds will be donated to the OWASP Foundation to the mobile security project for future use.
+Editing and proofreading by professional editors Graphic design and layout. Any leftover funds will be donated to the OWASP Foundation to the mobile security project for future use.
+
+## Smaller forms of donation
```

Comment:
I suggest putting _Other_ because write-contributions are not necessarily "small" and in many cases even more valuable as money contributions :)

```suggestion
## Other Forms of Donation
```